tests/cairo_files/instructions/test_environmental_information.cairo

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: MIT
+
+%lang starknet
+
+// Starkware dependencies
+from starkware.cairo.common.alloc import alloc
+from starkware.cairo.common.cairo_builtins import HashBuiltin, BitwiseBuiltin
+from starkware.cairo.common.uint256 import Uint256
+
+// Local dependencies
+from utils.utils import Helpers
+from kakarot.model import model
+from kakarot.stack import Stack
+from kakarot.constants import Constants
+from kakarot.execution_context import ExecutionContext
+from kakarot.instructions.environmental_information import EnvironmentalInformation
+
+@view
+func __setup__{
```

Comment:
what does this do?

Reply:
Sorry forgot to remove this when copying the format from the other unit tests 😢 I believe this is vestigial code from protostar testing, normally it acts as a hook that runs once before the tests run. Since this is not relevant in pytest, I'll remove them to make it cleaner 🙂

Reply:
thanks!